views/reports/custom_reports.php

```diff
@@ -20,161 +20,142 @@
-							<option value="1">January</option>
-							<option value="2">February</option>
-							<option value="3">March</option>
-							<option value="4">April</option>
-							<option value="5">May</option>
-							<option value="6">June</option>
-							<option value="7">July</option>
-							<option value="8">August</option>
-							<option value="9">September</option>
-							<option value="10">October</option>
-							<option value="11">November</option>
-							<option value="12">December</option>
-						</select>
-					</div>
+	<form action="custom-reports-table" method="POST" autocomplete="on">
+		<fieldset id="custom-reports-fields">
+			<!-- Select Basic -->
+			<div class="form-group row">
+				<label class="col-md-2 col-form-label" for="startDate"><b>Start Date</b></label>
+				<div class="col-md-4">
+					<input class="form-control" type="date" id="startDate" name="startDate" onchange="onStartDateChange()"/>
 				</div>
-				<!-- Select Basic -->
-				<div class="form-group row">
-					<label class="col-md-2 col-form-label" for="year"><b>Year</b></label>
-					<div class="col-md-4">
-						<select id="year" name="year" class="form-control">
-							<!-- YEAR OPTIONS INIT IN JAVASCRIPT BELOW.
-								BASED ON CURRENT YEAR.-->
-						</select>
-					</div>
+			</div>
+			<div class="form-group row">
+				<label class="col-md-2 col-form-label" for="endDate"><b>End Date</b></label>
+				<div class="col-md-4">
+					<input class="form-control" type="date" id="endDate" name="endDate" onchange="onEndDateChange()"/>
 				</div>
-				<!-- Multiple Checkboxes -->
-				<div class="form-group row">
-					<label class="col-md-2 col-form-label" for="location[]"><b>Location</b></label>
-					<div class="col-md-4">
-						<div class="checkbox">
-							<label for="location-0">
-							<input type="checkbox" name="location[]" id="location-0" value="Cornerstone">
-							Cornerstone
-							</label>
-						</div>
-						<div class="checkbox">
-							<label for="location-1">
-							<input type="checkbox" name="location[]" id="location-1" value="Dutchess County Jail">
-							Dutchess County Jail
-							</label>
-						</div>
-						<div class="checkbox">
-							<label for="location-2">
-							<input type="checkbox" name="location[]" id="location-2" value="Florence Manor">
-							Florence Manor
-							</label>
-						</div>
-						<div class="checkbox">
-							<label for="location-3">
-							<input type="checkbox" name="location[]" id="location-3" value="Fox Run">
-							Fox Run
-							</label>
-						</div>
-						<div class="checkbox">
-							<label for="location-4">
-							<input type="checkbox" name="location[]" id="location-4" value="ITAP Meadow Run">
-							ITAP Meadow Run
-							</label>
-						</div>
+			</div>
+			<!-- Multiple Checkboxes -->
+			<div id="custom-reports-checkboxes" class="form-group row">
+				<label class="col-md-2 col-form-label" for="location[]"><b>Location</b></label>
```

Comment:
This label should be renamed to `Curriculum`. The location is either "In-House", "Rehab", or "Jail". Also in the custom report table, make sure it says `Curricula` instead of "Locations" when displaying what I queried at the top.

Reply:
Sorry, so we're replacing Locations with Curriculum and replacing the options with "In-House", "Rehab", and "Jail"? Is that correct?

Reply:
Correct, although if possible, maybe pull the data from the enum in the database instead of statically configuring the options here. That way if the enum is changed this will be updated too. Not sure if that would mess with the queries but just a thought.

Reply:
Good point. For now, I'm just going to do it statically, but will try and pull the enums in the future.

Reply:
Sorry I misread this, the curricula will go here not the locations. So the options should be "Cornerstone", "Fox Run", etc.

Reply:
Gotcha, will fix.